GDJS/Runtime/variable.js

```diff
@@ -4,21 +4,36 @@
  * This project is released under the MIT License.
  */
 
+/**
+ * @typedef variableData Data representation of a GDevelop variable
+ * @property {string} [name] The name of the variable. Used if a child variable.
+ * @property {string | number} [value] The value of the variable, either string or number. Leave blank for structures.
+ * @property {variableData} [children] The children of the structure. Leave blank if value is defined.
+ */
+
 /**
  * A Variable is an object storing a value (number or a string) or children variables.
  *
  * @memberof gdjs
  * @class Variable
- * @param {Object} varData optional object used to initialize the variable.
+ * @param {Object} varData Optional object used to initialize the variable.
+ * @param {Array<variableData>} varData.value A representation of a variable.
```

Comment:
This seems wrong, there is only parameter of type `VariableData`:

`@param {VariableData} varData Optioanl object used to initialize the variable`

Reply:
Right, fixed that